### PR TITLE
istio conformance tutorial

### PR DESCRIPTION
**Pull request by kalantar**
@sriumcp you've make it so easy to add new tutorials :)

### CODE REVIEW

File: mkdocs/docs/tutorials/testing-patterns/conformance.md [hunk 1]
```diff
@@ -20,8 +20,64 @@ template: main.html
     
 Please follow steps 1, 2, and 3 of the [quick start tutorial](../../../getting-started/quick-start/#1-create-kubernetes-cluster).
 
-
 ## 4. Create app/ML model version
+=== "Istio"
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/testing-patterns/conformance.md:24]: This says "Deploy  a Knative app" while it should probably say "deploy an app?"

File: samples/istio/conformance/check.sh [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    echo "Packets were not lost"
+else
+    echo "Packets were lost"
+    echo "total requests:" $REQUESTSTOTAL
+    echo "200 requests:" $REQUESTS200
+    exit 1
+fi
+
+# Check if versionRecommendedForPromotion is productpage-v1; implies winner found
+expectedVrfp="productpage-v1"
```
Comment (sriumcp) [inline samples/istio/conformance/check.sh]: Could we please replace this by a check for the winner instead?

No  matter what happens, vrfp will be the baseline in the conformance test (but winner will depend on whether the baseline satisfied SLOs or not).

**Review by sriumcp — CHANGES_REQUESTED** [review]

File: mkdocs/docs/tutorials/testing-patterns/conformance.md (hunk 1 above)
Comment (kalantar) [reply mkdocs/docs/tutorials/testing-patterns/conformance.md:24]: done

File: samples/istio/conformance/check.sh (hunk 2 above)
Comment (kalantar) [reply samples/istio/conformance/check.sh]: done

**Comment by kalantar** [comment]
@sriumcp ready for review

File: mkdocs/docs/tutorials/testing-patterns/conformance.md [hunk 3]
```diff
@@ -49,13 +105,54 @@ Please follow steps 1, 2, and 3 of the [quick start tutorial](../../../getting-s
...
+        ```
+
 === "Knative"
     Generate requests using Fortio as follows.
 
     ```shell
     kubectl wait --for=condition=Ready ksvc/sample-app
     URL_VALUE=$(kubectl get ksvc sample-app -o json | jq .status.address.url)
-    sed "s+URL_VALUE+${URL_VALUE}+g" $ITER8/samples/knative/conformance/fortio.yaml | kubectl apply -f -
+    sed "s+URL_VALUE+${URL_VALUE}+g" $ITER8/samples/istio/conformance/fortio.yaml | kubectl apply -f -
```
Comment (sriumcp) [inline mkdocs/docs/tutorials/testing-patterns/conformance.md]: This shouldn't have been changed.

**Review by sriumcp — CHANGES_REQUESTED** [review]

File: mkdocs/docs/tutorials/testing-patterns/conformance.md (hunk 3 above)
Comment (kalantar) [reply mkdocs/docs/tutorials/testing-patterns/conformance.md]: agreed. fixed.

**Review by sriumcp — APPROVED** [review]
lgtm

**Pull request merged**